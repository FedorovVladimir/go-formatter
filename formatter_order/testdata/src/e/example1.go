package main

import "fmt"

var helloText = "hello" // want "formatter_order"

const name = "Vova" // want "formatter_order"

func main() { // want "formatter_order"
	fmt.Println(helloText, name)
}
