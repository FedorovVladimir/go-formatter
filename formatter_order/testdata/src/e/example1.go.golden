package main

import "fmt"

const name = "Vova" // want "formatter_order"

var helloText = "hello" // want "formatter_order"

func main() {
	fmt.Println(helloText, name)
}
